### ISSUE TITLE
Argument Type Check Error in Load-clustering exmaple

### ISSUE THREAD

**Issue by Roytangrb** [huseinzol05/Malaya#38]
Hi, curators of Malaya. I just started to explore this lib and encounter this issue while trying to run `/example/clustering/load-clustering.ipynb`

Error
![Screenshot 2020-03-15 at 8 59 47 PM](https://user-images.githubusercontent.com/25113108/76701854-f1582200-66ff-11ea-8419-f865b900729b.png)

was thrown when I tried to run 
```python
model = malaya.transformer.load(model = 'albert', size = 'base')
result_scatter = malaya.cluster.cluster_scatter(corpus,vectorizer=model)
```
where `model` suppose to be of string type.

Is this a bug? or am I missing something. Please advice help. 
Dependencies installed by running `build-dependencies.sh`

**Comment by huseinzol05** [comment]
oh, its a bug. thanks for let me know this!

**Comment by huseinzol05** [comment]
released 3.3.2!
